src/main/resources/migrations/00040_create_table_payouts.sql

```diff
@@ -0,0 +1,18 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:create_table_payouts
+CREATE TABLE payouts (
+    id BIGSERIAL PRIMARY KEY,
+    external_id VARCHAR(26),
+    amount BIGINT NOT NULL,
+    created_date TIMESTAMP WITH TIME ZONE DEFAULT (now() AT TIME ZONE 'utc') NOT NULL,
+    paid_out_date TIMESTAMP WITH TIME ZONE DEFAULT (now() AT TIME ZONE 'utc') NOT NULL,
```

Comment:
not sure default should be now() for paid_out_date. Any chance we don't get it from Stripe